.github/workflows/validate-pr.yaml

```diff
@@ -84,11 +81,14 @@ jobs:
                     base_url: https://github.com
           EOL
 
+      - name: Start containers
+        run: docker-compose up -d
```

Comment:
Oddly enough, the container seems to start faster and they were failing because /etc/monocle/config.yaml was not available.

Reply:
Yes that's not the first time I see that issue.